easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -18,12 +18,19 @@ builddependencies = [('zlib', '1.2.8')]
 preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 
-# make sure that system libraries are also considered by ld
-configopts = '--with-sysroot=/'
+# make sure that system libraries are also considered by ld and ld.gold is also built
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default '
 
-binlist = ['addr2line', 'ar', 'as', 'c++filt', 'elfedit', 'gprof', 'ld', 'ld.bfd', 'nm',
-           'objcopy', 'objdump', 'ranlib', 'readelf', 'size', 'strings', 'strip' ]
+# Ensure PIC
+buildopts = 'CFLAGS="-fPIC"'
```

Comment:
I'm pulling this from memory but I think this came about as a result of a need for libiberty.a to be PIC (maybe I should just include it in the commented section?)